### ISSUE TITLE
when adding measurement line, right click should cancel

### ISSUE THREAD

**Issue by ebrahimebrahim** [KitwareMedical/VolView#171]
If I start adding a measurement line and change my mind about where I placed the initial point, then at the moment I would have to place the second point and then click again to delete.

Based on other tools I've used that involve placing two points to define a line, I think it's intuitive to have right click cancel the line placement.

**Comment by floryst** [comment]
Good idea! In addition, I will also add support for hitting "esc" to cancel the current measurement.
